src/main/java/ingest/messaging/IngestThreadManager.java

```diff
@@ -57,169 +47,67 @@
  */
 @Component
 public class IngestThreadManager {
-	private static final String INGEST_TOPIC_NAME = IngestJob.class.getSimpleName();
 	@Autowired
 	private PiazzaLogger logger;
 	@Autowired
 	private IngestWorker ingestWorker;
 
-	@Value("${vcap.services.pz-kafka.credentials.host}")
-	private String KAFKA_HOSTS;
-	@Value("#{'${kafka.group}' + '-' + '${SPACE}'}")
-	private String KAFKA_GROUP;
 	@Value("${SPACE}")
 	private String SPACE;
 
-	private Producer<String, String> producer;
-	private Map<String, Future<?>> runningJobs;
-	private final AtomicBoolean closed = new AtomicBoolean(false);
-
+	private Map<String, Future<?>> runningJobs = new HashMap<String, Future<?>>();
 	private static final Logger LOG = LoggerFactory.getLogger(IngestThreadManager.class);
-	private static final String TOPIC_FORMAT = "%s-%s";
-
-	public IngestThreadManager() {
-		// Expected for Component instantiation
-	}
+	private ObjectMapper mapper = new ObjectMapper();
 
 	/**
-	 * Initializes the Thread Pool and begins Polling for Jobs.
-	 */
-	@PostConstruct
-	public void initialize() {
-		// Initialize the Kafka Producer
-		producer = KafkaClientFactory.getProducer(KAFKA_HOSTS);
-
-		// Log the initialization.
-		logger.log(String.format("Ingest listening to Kafka at %s in space %s.", KAFKA_HOSTS, SPACE), Severity.INFORMATIONAL);
-
-		// Initialize the Thread Pool and Map of running Threads
-		runningJobs = new HashMap<String, Future<?>>();
-
-		// Start polling for Kafka Jobs on the Group Consumer.. This occurs on a
-		// separate Thread so as not to block Spring.
-		Thread ingestJobsThread = new Thread() {
-			@Override
-			public void run() {
-				pollIngestJobs();
-			}
-		};
-		ingestJobsThread.start();
-
-		// Start polling for Kafka Abort Jobs on the unique Consumer.
-		Thread pollAbortThread = new Thread() {
-			@Override
-			public void run() {
-				pollAbortJobs();
-			}
-		};
-		pollAbortThread.start();
-	}
-
-	/**
-	 * Begins listening for Ingest Jobs.
+	 * Processes a message for Ingesting Data
+	 * 
+	 * @param ingestJobRequest
+	 *            The PiazzaJobRequest with the Ingest Job information
 	 */
-	public void pollIngestJobs() {
-		
-		Consumer<String, String> generalConsumer = null;
-		
+	@RabbitListener(queues = "IngestJob-${SPACE}")
```

Comment:
I see queues "IngestJob-SPACE" and "Abort-Job-SPACE". I would pick a naming convention and use it throughout (one has an additional hyphen).